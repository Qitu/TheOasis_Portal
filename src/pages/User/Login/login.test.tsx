import React from 'react';
import { act, fireEvent, render } from '@testing-library/react';
import { TestBrowser } from '@@/testBrowser'; // 根据实际路径进行导入
import { startMock } from '@@/requestRecordMock'; // 根据实际路径进行导入

let server: {
  close: () => void;
};

describe('Login Page', () => {
  beforeAll(async () => {
    server = await startMock({
      port: 8000,
      scene: 'login',
    });
  });

  afterAll(() => {
    server?.close();
  });

  it('should show login form', async () => {
    const historyRef = React.createRef<any>();
    const rootContainer = render(
      <TestBrowser
        historyRef={historyRef}
        location={{
          pathname: '/user/login',
        }}
      />,
    );

    await rootContainer.findAllByText('Oasis');

    act(() => {
      historyRef.current?.push('/user/login');
    });

    // 进行断言，根据您的需求添加更多断言
    // expect(rootContainer.baseElement?.querySelector('.ant-pro-form-login-desc')?.textContent).toBe(
    //   'Ant Design is the most influential web design specification in Xihu district',
    // );

    expect(rootContainer.asFragment()).toMatchSnapshot();

    await rootContainer.unmount();
  });
});
