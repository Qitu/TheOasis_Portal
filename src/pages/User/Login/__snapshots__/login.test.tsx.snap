// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Page should login success 1`] = `
<DocumentFragment>
  <div
    class="css-trkbkn"
  >
    <div
      style="flex: 1; padding: 36px 0px;"
    >
      <div
        class="ant-pro-form-login-container "
      >
        <div
          class="ant-pro-form-login-top"
        >
          <div
            class="ant-pro-form-login-header "
          >
            <span
              class="ant-pro-form-login-logo "
            >
              <img
                alt="logo"
                src="/logo2.png"
              />
            </span>
            <span
              class="ant-pro-form-login-title "
            >
              Oasis
            </span>
          </div>
        </div>
        <div
          class="ant-pro-form-login-main "
          style="width: 328px;"
        >
          <form
            autocomplete="off"
            class="ant-form ant-form-vertical ant-pro-form"
            id="auth"
          >
            <input
              style="display: none;"
              type="text"
            />
            <div
              class="ant-tabs ant-tabs-top ant-tabs-centered"
              style="padding: 8px;"
            >
              <div
                class="ant-tabs-nav"
                role="tablist"
              >
                <div
                  class="ant-tabs-nav-wrap"
                >
                  <div
                    class="ant-tabs-nav-list"
                    style="transform: translate(0px, 0px);"
                  >
                    <div
                      class="ant-tabs-tab ant-tabs-tab-active"
                      data-node-key="account"
                    >
                      <div
                        aria-controls="rc-tabs-test-panel-account"
                        aria-selected="true"
                        class="ant-tabs-tab-btn"
                        id="rc-tabs-test-tab-account"
                        role="tab"
                        tabindex="0"
                      >
                        Login
                      </div>
                    </div>
                    <div
                      class="ant-tabs-tab"
                      data-node-key="new"
                    >
                      <div
                        aria-controls="rc-tabs-test-panel-new"
                        aria-selected="false"
                        class="ant-tabs-tab-btn"
                        id="rc-tabs-test-tab-new"
                        role="tab"
                        tabindex="0"
                      >
                        Register
                      </div>
                    </div>
                    <div
                      class="ant-tabs-ink-bar ant-tabs-ink-bar-animated"
                      style="left: 0px; width: 0px;"
                    />
                  </div>
                </div>
                <div
                  class="ant-tabs-nav-operations ant-tabs-nav-operations-hidden"
                >
                  <button
                    aria-controls="rc-tabs-test-more-popup"
                    aria-expanded="false"
                    aria-haspopup="listbox"
                    aria-hidden="true"
                    class="ant-tabs-nav-more"
                    id="rc-tabs-test-more"
                    style="visibility: hidden; order: 1;"
                    tabindex="-1"
                    type="button"
                  >
                    <span
                      aria-label="ellipsis"
                      class="anticon anticon-ellipsis"
                      role="img"
                    >
                      <svg
                        aria-hidden="true"
                        data-icon="ellipsis"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="64 64 896 896"
                        width="1em"
                      >
                        <path
                          d="M176 511a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0z"
                        />
                      </svg>
                    </span>
                  </button>
                </div>
              </div>
              <div
                class="ant-tabs-content-holder"
              >
                <div
                  class="ant-tabs-content ant-tabs-content-top"
                >
                  <div
                    aria-hidden="false"
                    aria-labelledby="rc-tabs-test-tab-account"
                    class="ant-tabs-tabpane ant-tabs-tabpane-active"
                    id="rc-tabs-test-panel-account"
                    role="tabpanel"
                    tabindex="0"
                  />
                </div>
              </div>
            </div>
            <div
              class="ant-form-item ant-form-item-has-success"
            >
              <div
                class="ant-row ant-form-item-row"
              >
                <div
                  class="ant-col ant-form-item-control"
                >
                  <div
                    class="ant-form-item-control-input"
                  >
                    <div
                      class="ant-form-item-control-input-content"
                    >
                      <span
                        class="ant-input-affix-wrapper ant-input-affix-wrapper-lg ant-input-affix-wrapper-status-success"
                      >
                        <span
                          class="ant-input-prefix"
                        >
                          <span
                            aria-label="user"
                            class="anticon anticon-user"
                            role="img"
                          >
                            <svg
                              aria-hidden="true"
                              data-icon="user"
                              fill="currentColor"
                              focusable="false"
                              height="1em"
                              viewBox="64 64 896 896"
                              width="1em"
                            >
                              <path
                                d="M858.5 763.6a374 374 0 00-80.6-119.5 375.63 375.63 0 00-119.5-80.6c-.4-.2-.8-.3-1.2-.5C719.5 518 760 444.7 760 362c0-137-111-248-248-248S264 225 264 362c0 82.7 40.5 156 102.8 201.1-.4.2-.8.3-1.2.5-44.8 18.9-85 46-119.5 80.6a375.63 375.63 0 00-80.6 119.5A371.7 371.7 0 00136 901.8a8 8 0 008 8.2h60c4.4 0 7.9-3.5 8-7.8 2-77.2 33-149.5 87.8-204.3 56.7-56.7 132-87.9 212.2-87.9s155.5 31.2 212.2 87.9C779 752.7 810 825 812 902.2c.1 4.4 3.6 7.8 8 7.8h60a8 8 0 008-8.2c-1-47.8-10.9-94.3-29.5-138.2zM512 534c-45.9 0-89.1-17.9-121.6-50.4S340 407.9 340 362c0-45.9 17.9-89.1 50.4-121.6S466.1 190 512 190s89.1 17.9 121.6 50.4S684 316.1 684 362c0 45.9-17.9 89.1-50.4 121.6S557.9 534 512 534z"
                              />
                            </svg>
                          </span>
                        </span>
                        <input
                          class="ant-input ant-input-lg"
                          id="auth_username"
                          placeholder="Please enter your username"
                          type="text"
                          value="testtest"
                        />
                        <span
                          class="ant-input-suffix"
                        >
                          <span
                            class="ant-input-clear-icon"
                            role="button"
                            tabindex="-1"
                          >
                            <span
                              aria-label="close-circle"
                              class="anticon anticon-close-circle"
                              role="img"
                            >
                              <svg
                                aria-hidden="true"
                                data-icon="close-circle"
                                fill="currentColor"
                                fill-rule="evenodd"
                                focusable="false"
                                height="1em"
                                viewBox="64 64 896 896"
                                width="1em"
                              >
                                <path
                                  d="M512 64c247.4 0 448 200.6 448 448S759.4 960 512 960 64 759.4 64 512 264.6 64 512 64zm127.98 274.82h-.04l-.08.06L512 466.75 384.14 338.88c-.04-.05-.06-.06-.08-.06a.12.12 0 00-.07 0c-.03 0-.05.01-.09.05l-45.02 45.02a.2.2 0 00-.05.09.12.12 0 000 .07v.02a.27.27 0 00.06.06L466.75 512 338.88 639.86c-.05.04-.06.06-.06.08a.12.12 0 000 .07c0 .03.01.05.05.09l45.02 45.02a.2.2 0 00.09.05.12.12 0 00.07 0c.02 0 .04-.01.08-.05L512 557.25l127.86 127.87c.04.04.06.05.08.05a.12.12 0 00.07 0c.03 0 .05-.01.09-.05l45.02-45.02a.2.2 0 00.05-.09.12.12 0 000-.07v-.02a.27.27 0 00-.05-.06L557.25 512l127.87-127.86c.04-.04.05-.06.05-.08a.12.12 0 000-.07c0-.03-.01-.05-.05-.09l-45.02-45.02a.2.2 0 00-.09-.05.12.12 0 00-.07 0z"
                                />
                              </svg>
                            </span>
                          </span>
                        </span>
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="ant-form-item ant-form-item-has-success"
            >
              <div
                class="ant-row ant-form-item-row"
              >
                <div
                  class="ant-col ant-form-item-control"
                >
                  <div
                    class="ant-form-item-control-input"
                  >
                    <div
                      class="ant-form-item-control-input-content"
                    >
                      <span
                        class="ant-input-affix-wrapper ant-input-password ant-input-password-large ant-input-affix-wrapper-lg ant-input-affix-wrapper-status-success"
                      >
                        <span
                          class="ant-input-prefix"
                        >
                          <span
                            aria-label="lock"
                            class="anticon anticon-lock"
                            role="img"
                          >
                            <svg
                              aria-hidden="true"
                              data-icon="lock"
                              fill="currentColor"
                              focusable="false"
                              height="1em"
                              viewBox="64 64 896 896"
                              width="1em"
                            >
                              <path
                                d="M832 464h-68V240c0-70.7-57.3-128-128-128H388c-70.7 0-128 57.3-128 128v224h-68c-17.7 0-32 14.3-32 32v384c0 17.7 14.3 32 32 32h640c17.7 0 32-14.3 32-32V496c0-17.7-14.3-32-32-32zM332 240c0-30.9 25.1-56 56-56h248c30.9 0 56 25.1 56 56v224H332V240zm460 600H232V536h560v304zM484 701v53c0 4.4 3.6 8 8 8h40c4.4 0 8-3.6 8-8v-53a48.01 48.01 0 10-56 0z"
                              />
                            </svg>
                          </span>
                        </span>
                        <input
                          class="ant-input ant-input-lg"
                          id="auth_password"
                          placeholder="Please enter your password"
                          type="password"
                        />
                        <span
                          class="ant-input-suffix"
                        >
                          <span
                            aria-label="eye-invisible"
                            class="anticon anticon-eye-invisible ant-input-password-icon"
                            role="img"
                            tabindex="-1"
                          >
                            <svg
                              aria-hidden="true"
                              data-icon="eye-invisible"
                              fill="currentColor"
                              focusable="false"
                              height="1em"
                              viewBox="64 64 896 896"
                              width="1em"
                            >
                              <path
                                d="M942.2 486.2Q889.47 375.11 816.7 305l-50.88 50.88C807.31 395.53 843.45 447.4 874.7 512 791.5 684.2 673.4 766 512 766q-72.67 0-133.87-22.38L323 798.75Q408 838 512 838q288.3 0 430.2-300.3a60.29 60.29 0 000-51.5zm-63.57-320.64L836 122.88a8 8 0 00-11.32 0L715.31 232.2Q624.86 186 512 186q-288.3 0-430.2 300.3a60.3 60.3 0 000 51.5q56.69 119.4 136.5 191.41L112.48 835a8 8 0 000 11.31L155.17 889a8 8 0 0011.31 0l712.15-712.12a8 8 0 000-11.32zM149.3 512C232.6 339.8 350.7 258 512 258c54.54 0 104.13 9.36 149.12 28.39l-70.3 70.3a176 176 0 00-238.13 238.13l-83.42 83.42C223.1 637.49 183.3 582.28 149.3 512zm246.7 0a112.11 112.11 0 01146.2-106.69L401.31 546.2A112 112 0 01396 512z"
                              />
                              <path
                                d="M508 624c-3.46 0-6.87-.16-10.25-.47l-52.82 52.82a176.09 176.09 0 00227.42-227.42l-52.82 52.82c.31 3.38.47 6.79.47 10.25a111.94 111.94 0 01-112 112z"
                              />
                            </svg>
                          </span>
                        </span>
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              style="margin-bottom: 24px;"
            >
              <label
                class="ant-checkbox-wrapper ant-checkbox-wrapper-checked"
              >
                <span
                  class="ant-checkbox ant-wave-target ant-checkbox-checked"
                >
                  <input
                    checked=""
                    class="ant-checkbox-input"
                    id="auth_autoLogin"
                    type="checkbox"
                  />
                  <span
                    class="ant-checkbox-inner"
                  />
                </span>
                <span>
                  Auto Login
                </span>
              </label>
              <a
                style="float: right;"
              >
                Don't have account?
              </a>
            </div>
            <button
              class="ant-btn ant-btn-primary ant-btn-lg"
              style="width: 100%;"
              type="button"
            >
              <span>
                Login
              </span>
            </button>
          </form>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Login Page should show login form 1`] = `
<DocumentFragment>
  <div
    class="css-trkbkn"
  >
    <div
      style="flex: 1; padding: 36px 0px;"
    >
      <div
        class="ant-pro-form-login-container "
      >
        <div
          class="ant-pro-form-login-top"
        >
          <div
            class="ant-pro-form-login-header "
          >
            <span
              class="ant-pro-form-login-logo "
            >
              <img
                alt="logo"
                src="/logo2.png"
              />
            </span>
            <span
              class="ant-pro-form-login-title "
            >
              Oasis
            </span>
          </div>
        </div>
        <div
          class="ant-pro-form-login-main "
          style="width: 328px;"
        >
          <form
            autocomplete="off"
            class="ant-form ant-form-vertical ant-pro-form"
            id="auth"
          >
            <input
              style="display: none;"
              type="text"
            />
            <div
              class="ant-tabs ant-tabs-top ant-tabs-centered"
              style="padding: 8px;"
            >
              <div
                class="ant-tabs-nav"
                role="tablist"
              >
                <div
                  class="ant-tabs-nav-wrap"
                >
                  <div
                    class="ant-tabs-nav-list"
                    style="transform: translate(0px, 0px);"
                  >
                    <div
                      class="ant-tabs-tab ant-tabs-tab-active"
                      data-node-key="account"
                    >
                      <div
                        aria-controls="rc-tabs-test-panel-account"
                        aria-selected="true"
                        class="ant-tabs-tab-btn"
                        id="rc-tabs-test-tab-account"
                        role="tab"
                        tabindex="0"
                      >
                        Login
                      </div>
                    </div>
                    <div
                      class="ant-tabs-tab"
                      data-node-key="new"
                    >
                      <div
                        aria-controls="rc-tabs-test-panel-new"
                        aria-selected="false"
                        class="ant-tabs-tab-btn"
                        id="rc-tabs-test-tab-new"
                        role="tab"
                        tabindex="0"
                      >
                        Register
                      </div>
                    </div>
                    <div
                      class="ant-tabs-ink-bar ant-tabs-ink-bar-animated"
                    />
                  </div>
                </div>
                <div
                  class="ant-tabs-nav-operations ant-tabs-nav-operations-hidden"
                >
                  <button
                    aria-controls="rc-tabs-test-more-popup"
                    aria-expanded="false"
                    aria-haspopup="listbox"
                    aria-hidden="true"
                    class="ant-tabs-nav-more"
                    id="rc-tabs-test-more"
                    style="visibility: hidden; order: 1;"
                    tabindex="-1"
                    type="button"
                  >
                    <span
                      aria-label="ellipsis"
                      class="anticon anticon-ellipsis"
                      role="img"
                    >
                      <svg
                        aria-hidden="true"
                        data-icon="ellipsis"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="64 64 896 896"
                        width="1em"
                      >
                        <path
                          d="M176 511a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0z"
                        />
                      </svg>
                    </span>
                  </button>
                </div>
              </div>
              <div
                class="ant-tabs-content-holder"
              >
                <div
                  class="ant-tabs-content ant-tabs-content-top"
                >
                  <div
                    aria-hidden="false"
                    aria-labelledby="rc-tabs-test-tab-account"
                    class="ant-tabs-tabpane ant-tabs-tabpane-active"
                    id="rc-tabs-test-panel-account"
                    role="tabpanel"
                    tabindex="0"
                  />
                </div>
              </div>
            </div>
            <div
              class="ant-form-item"
            >
              <div
                class="ant-row ant-form-item-row"
              >
                <div
                  class="ant-col ant-form-item-control"
                >
                  <div
                    class="ant-form-item-control-input"
                  >
                    <div
                      class="ant-form-item-control-input-content"
                    >
                      <span
                        class="ant-input-affix-wrapper ant-input-affix-wrapper-lg"
                      >
                        <span
                          class="ant-input-prefix"
                        >
                          <span
                            aria-label="user"
                            class="anticon anticon-user"
                            role="img"
                          >
                            <svg
                              aria-hidden="true"
                              data-icon="user"
                              fill="currentColor"
                              focusable="false"
                              height="1em"
                              viewBox="64 64 896 896"
                              width="1em"
                            >
                              <path
                                d="M858.5 763.6a374 374 0 00-80.6-119.5 375.63 375.63 0 00-119.5-80.6c-.4-.2-.8-.3-1.2-.5C719.5 518 760 444.7 760 362c0-137-111-248-248-248S264 225 264 362c0 82.7 40.5 156 102.8 201.1-.4.2-.8.3-1.2.5-44.8 18.9-85 46-119.5 80.6a375.63 375.63 0 00-80.6 119.5A371.7 371.7 0 00136 901.8a8 8 0 008 8.2h60c4.4 0 7.9-3.5 8-7.8 2-77.2 33-149.5 87.8-204.3 56.7-56.7 132-87.9 212.2-87.9s155.5 31.2 212.2 87.9C779 752.7 810 825 812 902.2c.1 4.4 3.6 7.8 8 7.8h60a8 8 0 008-8.2c-1-47.8-10.9-94.3-29.5-138.2zM512 534c-45.9 0-89.1-17.9-121.6-50.4S340 407.9 340 362c0-45.9 17.9-89.1 50.4-121.6S466.1 190 512 190s89.1 17.9 121.6 50.4S684 316.1 684 362c0 45.9-17.9 89.1-50.4 121.6S557.9 534 512 534z"
                              />
                            </svg>
                          </span>
                        </span>
                        <input
                          class="ant-input ant-input-lg"
                          id="auth_username"
                          placeholder="Please enter your username"
                          type="text"
                          value=""
                        />
                        <span
                          class="ant-input-suffix"
                        >
                          <span
                            class="ant-input-clear-icon ant-input-clear-icon-hidden"
                            role="button"
                            tabindex="-1"
                          >
                            <span
                              aria-label="close-circle"
                              class="anticon anticon-close-circle"
                              role="img"
                            >
                              <svg
                                aria-hidden="true"
                                data-icon="close-circle"
                                fill="currentColor"
                                fill-rule="evenodd"
                                focusable="false"
                                height="1em"
                                viewBox="64 64 896 896"
                                width="1em"
                              >
                                <path
                                  d="M512 64c247.4 0 448 200.6 448 448S759.4 960 512 960 64 759.4 64 512 264.6 64 512 64zm127.98 274.82h-.04l-.08.06L512 466.75 384.14 338.88c-.04-.05-.06-.06-.08-.06a.12.12 0 00-.07 0c-.03 0-.05.01-.09.05l-45.02 45.02a.2.2 0 00-.05.09.12.12 0 000 .07v.02a.27.27 0 00.06.06L466.75 512 338.88 639.86c-.05.04-.06.06-.06.08a.12.12 0 000 .07c0 .03.01.05.05.09l45.02 45.02a.2.2 0 00.09.05.12.12 0 00.07 0c.02 0 .04-.01.08-.05L512 557.25l127.86 127.87c.04.04.06.05.08.05a.12.12 0 00.07 0c.03 0 .05-.01.09-.05l45.02-45.02a.2.2 0 00.05-.09.12.12 0 000-.07v-.02a.27.27 0 00-.05-.06L557.25 512l127.87-127.86c.04-.04.05-.06.05-.08a.12.12 0 000-.07c0-.03-.01-.05-.05-.09l-45.02-45.02a.2.2 0 00-.09-.05.12.12 0 00-.07 0z"
                                />
                              </svg>
                            </span>
                          </span>
                        </span>
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="ant-form-item"
            >
              <div
                class="ant-row ant-form-item-row"
              >
                <div
                  class="ant-col ant-form-item-control"
                >
                  <div
                    class="ant-form-item-control-input"
                  >
                    <div
                      class="ant-form-item-control-input-content"
                    >
                      <span
                        class="ant-input-affix-wrapper ant-input-password ant-input-password-large ant-input-affix-wrapper-lg"
                      >
                        <span
                          class="ant-input-prefix"
                        >
                          <span
                            aria-label="lock"
                            class="anticon anticon-lock"
                            role="img"
                          >
                            <svg
                              aria-hidden="true"
                              data-icon="lock"
                              fill="currentColor"
                              focusable="false"
                              height="1em"
                              viewBox="64 64 896 896"
                              width="1em"
                            >
                              <path
                                d="M832 464h-68V240c0-70.7-57.3-128-128-128H388c-70.7 0-128 57.3-128 128v224h-68c-17.7 0-32 14.3-32 32v384c0 17.7 14.3 32 32 32h640c17.7 0 32-14.3 32-32V496c0-17.7-14.3-32-32-32zM332 240c0-30.9 25.1-56 56-56h248c30.9 0 56 25.1 56 56v224H332V240zm460 600H232V536h560v304zM484 701v53c0 4.4 3.6 8 8 8h40c4.4 0 8-3.6 8-8v-53a48.01 48.01 0 10-56 0z"
                              />
                            </svg>
                          </span>
                        </span>
                        <input
                          class="ant-input ant-input-lg"
                          id="auth_password"
                          placeholder="Please enter your password"
                          type="password"
                          value=""
                        />
                        <span
                          class="ant-input-suffix"
                        >
                          <span
                            aria-label="eye-invisible"
                            class="anticon anticon-eye-invisible ant-input-password-icon"
                            role="img"
                            tabindex="-1"
                          >
                            <svg
                              aria-hidden="true"
                              data-icon="eye-invisible"
                              fill="currentColor"
                              focusable="false"
                              height="1em"
                              viewBox="64 64 896 896"
                              width="1em"
                            >
                              <path
                                d="M942.2 486.2Q889.47 375.11 816.7 305l-50.88 50.88C807.31 395.53 843.45 447.4 874.7 512 791.5 684.2 673.4 766 512 766q-72.67 0-133.87-22.38L323 798.75Q408 838 512 838q288.3 0 430.2-300.3a60.29 60.29 0 000-51.5zm-63.57-320.64L836 122.88a8 8 0 00-11.32 0L715.31 232.2Q624.86 186 512 186q-288.3 0-430.2 300.3a60.3 60.3 0 000 51.5q56.69 119.4 136.5 191.41L112.48 835a8 8 0 000 11.31L155.17 889a8 8 0 0011.31 0l712.15-712.12a8 8 0 000-11.32zM149.3 512C232.6 339.8 350.7 258 512 258c54.54 0 104.13 9.36 149.12 28.39l-70.3 70.3a176 176 0 00-238.13 238.13l-83.42 83.42C223.1 637.49 183.3 582.28 149.3 512zm246.7 0a112.11 112.11 0 01146.2-106.69L401.31 546.2A112 112 0 01396 512z"
                              />
                              <path
                                d="M508 624c-3.46 0-6.87-.16-10.25-.47l-52.82 52.82a176.09 176.09 0 00227.42-227.42l-52.82 52.82c.31 3.38.47 6.79.47 10.25a111.94 111.94 0 01-112 112z"
                              />
                            </svg>
                          </span>
                        </span>
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              style="margin-bottom: 24px;"
            >
              <label
                class="ant-checkbox-wrapper ant-checkbox-wrapper-checked"
              >
                <span
                  class="ant-checkbox ant-wave-target ant-checkbox-checked"
                >
                  <input
                    checked=""
                    class="ant-checkbox-input"
                    id="auth_autoLogin"
                    type="checkbox"
                  />
                  <span
                    class="ant-checkbox-inner"
                  />
                </span>
                <span>
                  Auto Login
                </span>
              </label>
              <a
                style="float: right;"
              >
                Don't have account?
              </a>
            </div>
            <button
              class="ant-btn ant-btn-primary ant-btn-lg"
              style="width: 100%;"
              type="button"
            >
              <span>
                Login
              </span>
            </button>
          </form>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
